http/src/main/resources/reference.conf

```diff
@@ -27,7 +27,7 @@ dataproc {
   #vpcSubnet = "default"
   # This image is used for legacy jupyter image where hail is compatible with earlier version of spark
   legacyCustomDataprocImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-image-dbb0929"
```

Comment:
Should this job also update `legacyCustomDataprocImage` to `custom-leo-image-1-2-79-debian9-0c0865b`?

https://console.cloud.google.com/compute/images?project=broad-dsp-gcr-public&tab=images&imagessize=50&imagessort=-creationTimestamp%252CdefaultSortKey&imagesquery=%255B%257B_22k_22_3A_22_22_2C_22t_22_3A10_2C_22v_22_3A_22_5C_22leo_5C_22_22%257D%255D

Reply:
it definitely should...forgot to create a ticket when I noticed this